.github/workflows/trigger-dag-reuse-wf.yaml

```diff
@@ -9,11 +9,6 @@ on:  # yamllint disable-line rule:truthy
         required: false
         type: string
         default: ''
-      wait_time:
```

Comment:
I thought we still need this wait time for the pods to be ready. Or does  the `HEALTHY` state indicate that the whole deployment is ready?

Reply:
@Lee-W this core API endpoint check pods health only and then only status is returned as HEALTHY